lib/bolt_server/transport_app.rb

```diff
@@ -305,7 +305,13 @@ def make_winrm_target(target_hash)
     # @param environment [String] the environment to fetch the plan from
     get '/plans/:module_name/:plan_name' do
       in_pe_pal_env(params['environment']) do |pal|
-        plan_info = pal.get_plan_info("#{params[:module_name]}::#{params[:plan_name]}")
+        # Handle case where plan name is simply module name with special `init.pp` plan
+        plan_name = if params[:plan_name] == 'init'
```

Comment:
Maybe this is straightforward enough not to test, but it also probably couldn't hurt? https://github.com/puppetlabs/bolt/blob/master/spec/bolt_server/transport_app_spec.rb#L73

Reply:
Yeah, for sure. Found this bug working on orchestrator-client and pushed a quick fix to show Sean the problem. Added a test :)